client/landing/stepper/declarative-flow/test/site-migration-flow.tsx

```diff
@@ -20,12 +20,17 @@ describe( 'Site Migration Flow', () => {
 		Object.defineProperty( window, 'location', originalLocation );
 	} );
 
+	beforeEach( () => {
```

Comment:
Why is this needed as part of this PR? or this is a fix for something that was happening previously?

I know that there's a reason, but I would like to understand it :).

Reply:
I'm assuming that it's erasing any previously configured mock functions so each test starts from a clean place.

But I'm also curious if this is fixing an issue or just an overall improvement.

Reply:
> But I'm also curious if this is fixing an issue or just an overall improvement.
I would say a bit of both. 

The original cause was that my test was passing incorrectly because without it and adding an "from=[YOUR_SITE]" will make the test pass while I am not passing the from param.